client/components/settings/settingBody.js

```diff
@@ -0,0 +1,132 @@
+Meteor.subscribe('setting');
+Meteor.subscribe('mailServer');
+
+BlazeComponent.extendComponent({
+  onCreated() {
+    this.error = new ReactiveVar('');
+    this.loading = new ReactiveVar(false);
+    this.generalSetting = new ReactiveVar(true);
+    this.emailSetting = new ReactiveVar(false);
+  },
+
+  setError(error) {
+    this.error.set(error);
+  },
+
+  setLoading(w) {
+    this.loading.set(w);
+  },
+
+  checkField(selector) {
+    const value = $(selector).val();
+    if(!value || value.trim() === ''){
+      $(selector).parent().parent().addClass('has-error');
+      throw Error('blank field');
+    } else {
+      return value;
+    }
+  },
+
+  currentSetting(){
+    return Settings.findOne();
+  },
+
+  boards() {
+    return Boards.find({
+      archived: false,
+      'members.userId': Meteor.userId(),
+      'members.isAdmin': true,
+    }, {
+      sort: ['title'],
+    });
+  },
+  toggleStrictMode(){
+    this.setLoading(true);
+    const isStrictMode = this.currentSetting().strict;
```

Comment:
Um, my original thought was there is a bunch of other "more strict functionalities" in strict mode, not only invitationOnly.